sechub-doc/src/test/java/com/daimler/sechub/restdoc/ProjectAdministrationRestControllerRestDocTest.java

```diff
@@ -273,19 +280,86 @@ public void restdoc_show_project_details() throws Exception {
 							fieldWithPath(ProjectDetailInformation.PROPERTY_WHITELIST).description("A list of all whitelisted URIs. Only these ones can be scanned for the project!"),
 							fieldWithPath(ProjectDetailInformation.PROPERTY_METADATA).description("An JSON object containing metadata key-value pairs defined for this project."),
-							fieldWithPath(ProjectDetailInformation.PROPERTY_METADATA + ".key1").description("An arbitrary metadata key")
+							fieldWithPath(ProjectDetailInformation.PROPERTY_METADATA + ".key1").description("An arbitrary metadata key."),
+							fieldWithPath(ProjectDetailInformation.PROPERTY_DESCRIPTION).description("The project description.")
 						)
 					)
 				);
 
 		/* @formatter:on */
     }
 
+    @Test
+    @UseCaseRestDoc(useCase = UseCaseAdministratorChangesProjectDetails.class)
+    public void restdoc_change_project_description() throws Exception {
+        /* prepare */
+        Project project = mock(Project.class);
+        when(project.getId()).thenReturn("projectId1");
+
+        Set<User> users = new LinkedHashSet<>();
+        User user1 = mock(User.class);
+        when(user1.getName()).thenReturn("name1");
+
+        User user2 = mock(User.class);
+        when(user2.getName()).thenReturn("name2");
+
+        users.add(user1);
+        users.add(user2);
+
+        when(project.getUsers()).thenReturn(users);
+        when(project.getOwner()).thenReturn(user1);
+        Set<URI> whiteList = new LinkedHashSet<>();
+        whiteList.add(new URI("http://www.sechub.example.org"));
+        when(project.getWhiteList()).thenReturn(whiteList);
+
+        Set<ProjectMetaDataEntity> metaData = new LinkedHashSet<>();
+        ProjectMetaDataEntity entry = new ProjectMetaDataEntity("projectId1", "key1", "value1");
+        metaData.add(entry);
+
+        when(project.getMetaData()).thenReturn(metaData);
+        
+        when(project.getDescription()).thenReturn("description");
+
+        ProjectDetailInformation detailInformation = new ProjectDetailInformation(project);
+        
+        when(detailsChangeService.changeDetails(any(), any())).thenReturn(detailInformation);
+
+        /* execute + test @formatter:off */
+        this.mockMvc.perform(
+                put(https(PORT_USED).buildAdminChangesProjectDescriptionUrl(PROJECT_ID.pathElement()), "projectId1").
+                content("{\n"
+                        + "  \"name\" : \"projectId1\", \n"
+                        + "  \"description\" : \"new description\"\n"
+                        + "}").
+                contentType(MediaType.APPLICATION_JSON_VALUE)
+                )./*
+                */
+        andDo(print()).
+        andExpect(status().isOk()).
+        andDo(document(RestDocPathFactory.createPath(UseCaseAdministratorChangesProjectDetails.class),
+                pathParameters(
+                            parameterWithName(PROJECT_ID.paramName()).description("The id for project to show details for")
```

Comment:
I think this is a copy and past mistake: `The id for project to show details for` should be `The id for project to change details for`.

Reply:
done